### ISSUE TITLE
Duplication of CREATE TABLE messenger_messages instructions in diff when messenger transport config auto_setup: false

### ISSUE THREAD

**Issue by florianajir** [jsor/doctrine-postgis#55]
Initially posted on the wrong repo https://github.com/symfony/symfony/issues/47912 my issue is related to this package. It causes duplication of messenger_messages table creation on doctrine schema (or migration) diff 

Lines duplicated:
```
CREATE TABLE messenger_messages (id BIGSERIAL NOT NULL, body TEXT NOT NULL, headers TEXT NOT NULL, queue_name VARCHAR(190) NOT NULL, created_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, available_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, delivered_at TIMESTAMP(0) WITHOUT TIME ZONE DEFAULT NULL, PRIMARY KEY(id));
CREATE INDEX IDX_75EA56E0FB7336F0 ON messenger_messages (queue_name);
CREATE INDEX IDX_75EA56E0E3BD61CE ON messenger_messages (available_at);
CREATE INDEX IDX_75EA56E016BA31DB ON messenger_messages (delivered_at);
```

To reproduce add these packages to a symfony project composer file:
```
"jsor/doctrine-postgis": "^2.1",
"doctrine/doctrine-bundle": "^2.1",
"doctrine/doctrine-migrations-bundle": "^3.0",
"doctrine/orm": "^2.9",
"symfony/doctrine-messenger": "6.1.*",
```

In `messenger.yaml` config file set your transport with `auto_setup: false` 
(doc: https://symfony.com/doc/current/messenger.html#transport-configuration)

run `php bin/console doctrine:migra:diff --from-empty-schema` or ` php bin/console doctrine:sch:up --dump-sql`

Check the content generated (duplication)

**Comment by florianajir** [comment]
If this can help I debugged the script and this screenshots shows the dual call to getCreateTableSQL
![Capture d’écran 2022-10-26 201652](https://user-images.githubusercontent.com/3715903/198115942-b92dfe74-8951-41b4-856e-0d9b3727d41d.jpg)
![Capture d’écran 2022-10-26 201736](https://user-images.githubusercontent.com/3715903/198115955-bd02cd10-cf40-4752-b91a-f4844bf7d20a.jpg)

**Comment by jsor** [comment]
This should be fixed with the latest [v2.2.0 release](https://github.com/jsor/doctrine-postgis/releases/tag/v2.2.0)
